lib/workers/branch/index.ts

```diff
@@ -442,10 +448,7 @@ export async function processBranch(
         logger.debug('Branch is automerged - returning');
         return { branchExists: false, result: BranchResult.Automerged };
       }
-      if (
-        mergeStatus === 'stale' &&
-        ['conflicted', 'never'].includes(config.rebaseWhen)
-      ) {
+      if (mergeStatus === 'stale' && config.rebaseWhen === 'conflicted') {
         logger.warn(
           'Branch cannot automerge because it is stale and rebaseWhen setting disallows rebasing - raising a PR instead'
         );
```

Comment:
This looks wrong to me. 🤔

Reply:
Reverted this change. Or did you mean the whole condition?

Reply:
Yeah, the condition checks for conflict and the message says never. So they didn't match for me.